pxtblocks/code-validation/runValidatorPlanAsync.ts

```diff
@@ -1,11 +1,14 @@
 namespace pxt.blocks {
+
```

Comment:
@srietkerk The changes in this file are responding to your comments in https://github.com/microsoft/pxt/pull/9835. They're not related to the indexed db change, I actually forgot I'd made them in this branch, but I suppose here they are 😅